common/app/views/fragments/email/signup/emailIframe.scala.html

```diff
@@ -2,4 +2,11 @@
     campaignCode: String,
     formType: String = "footer")
 
-<iframe title="Guardian Email Sign-up Form" src="@{"/email/form/" + formType + "/" + listName}" scrolling="no" seamless id="@{formType + "__email-form"}" frameborder="0" class="iframed--overflow-hidden email-sub__iframe js-email-sub__iframe" data-form-success-desc="We will send you our picks of the most important headlines tomorrow morning."></iframe>
+<iframe
+    title="Guardian Email Sign-up Form"
+    src="@{"/email/form/" + formType + "/" + listName}"
+    scrolling="no" seamless
+    id="@{formType + "__email-form"}" frameborder="0"
+    class="iframed--overflow-hidden email-sub__iframe js-email-sub__iframe"
+    data-form-success-headline="We've sent you an email!"
+    data-form-success-desc="Open up the email and click the verify button to confirm your email address.<br/><br/>You'll then start receiving the daily email."></iframe>
```

Comment:
I'm not a fan of the double `br` as you've mentioned, but equally, it was there before so you're not making it worse!